src/Task.test.js

```diff
@@ -0,0 +1,31 @@
+import { Tasks } from './task.js';
+/* eslint-disable */
+describe('Tasks', () => {
```

Comment:
- [ ] Please remove the eslint disablement line and fix the linters errors by yourself because linters is a great tool to help you write professional, clean, and readable code.